faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -163,6 +165,16 @@ object DateV {
   def apply(value: String): DateV = DateV(LocalDate.parse(value))
 }
 
+case class BytesV(@(JsonIgnore @param @field @getter) bytes: IndexedSeq[Byte]) extends ScalarValue {
+  @JsonProperty("@bytes")
+  lazy val strValue = BaseEncoding.base64Url().encode(bytes.toArray)
+}
+object BytesV {
+  def apply(bytes: Array[Byte]): BytesV = BytesV(bytes.toIndexedSeq)
+  def apply(bytes: Int*): BytesV = BytesV(bytes map { _.toByte } toIndexedSeq)
```

Comment:
it was just a sugar I added, because values above `0x80` are not considered bytes anymore and is treated as int only, so things like this `Bytes(0x80, 0x90, 0xff)` wouldn't be possible